test.js

```diff
@@ -32,8 +33,7 @@ test('unherit(Super)', function (t) {
 
   var B = unherit(A)
 
-  // eslint-disable-next-line new-cap
-  var b = B('foo', 'bar', 'baz')
+  var b = new B('foo', 'bar', 'baz')
```

Comment:
this could alternatively be TS ignored

Reply:
this *might* be testing when the class is used w/o `new`, and then it’s instantiated by the instanceof check in `Of`